src/components/overlays/AddMultipleUser/index.tsx

```diff
@@ -108,11 +113,24 @@ export default function AddMultipleUser() {
   }, [idpsData])
 
   useEffect(() => {
-    uploadAPIRResponse &&
+    if (uploadAPIRResponse) {
+      const errorMsgs = uploadAPIRResponse.errors.map(
+        (error: ErrorResponse) => {
+          return [
+            csvData[error.line][0] +
+              ' ' +
+              csvData[error.line][1] +
+              ', ' +
+              csvData[error.line][2],
+            error.message,
+          ]
+        }
```

Comment:
please use a template string for this - also I think we don't need return if we wrap the array in round brackets:
`(error: ErrorResponse) => (['${...} ${...}, ${...}', ...])`
(replace ' with backticks)

Reply:
@oyo Made the change